PeerioTechnologies/peerio-desktop
# chore: Electron 3.0

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/13900/electron-3-0

- [x] Update Electron
- [x] Rebuild node-keytar 
- [x] Fix extensions

#### Testing instructions  

* Check that every platform can load and store account key (i.e. automatic sign in works). There should be no errors in console about `keytar`.
* TBD


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
